src/libraries/Microsoft.PowerFx.Core/Public/Config/RegexCache.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx
+{
+    internal class RegexCache
```

Comment:
>RegexCache [](http://example.com/codeflow?start=19&length=10)

nit - since this is just for compile time (not runtime), can we rename, may be to RegexTypeCache.
And adda comment.